docs/quickstarts/react.mdx

```diff
@@ -16,18 +15,15 @@ description: Learn how to use Clerk to quickly and easily add secure authenticat
     {
       title: "Set up a Clerk application",
       link: "https://clerk.com/docs/quickstarts/setup-clerk"
-    },
-    {
-      title: "Create a React application using Vite",
-      link: "https://vitejs.dev/guide/#scaffolding-your-first-vite-project"
     }
   ]}
 >
+- Create a new React application using Vite
 - Install `@clerk/clerk-react`
 - Set up your environment keys
 - Import the Publishable Key
 - Wrap your React app in `<ClerkProvider />`
-- Protecting your content
+- Use Clerk's control components to protect your content
```

Comment:
"Control components" is an old holdover of internal jargon. Recommend just "components"